<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analayzer</class>
 <widget class="QDialog" name="analayzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analuzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Начальная частота:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Конечная частота:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Кол-во частотоных точек:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="averag_count">
         <property name="text">
          <string>Количество усреднений:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Мощность:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="start_freq"/>
       </item>
       <item>
        <widget class="QLineEdit" name="end_freq"/>
       </item>
       <item>
        <widget class="QLineEdit" name="step_freq"/>
       </item>
       <item>
        <widget class="QLineEdit" name="avareg_count"/>
       </item>
       <item>
        <widget class="QLineEdit" name="power"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Connect_btn">
       <property name="text">
        <string>Подключится</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnecnt_btn">
       <property name="text">
        <string>Отключится</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
